### PR TITLE
CI: Use latest run_task version

### PR DESCRIPTION
**Pull request by tcharding**
Recently we discovered a bug in the `run_task` script where we forgot to use `--no-default-features` in a subset of test runs (the feature matrix stuff).

Upgrade to use the latest version of `run_task.sh`.

### CODE REVIEW

**Comment by tcharding** [comment]
Odd, this is using `unicode-width 0.1.14` which was only released 6 hours ago, smells like a bug in CI but `run_task` looks like it always uses `cargo --locked`?

**Comment by apoelstra** [comment]
@tcharding it's the schemars tests. They do not have lockfiles.

I vote we just kill these tests and drop schemars.

**Comment by tcharding** [comment]
Needs #3395

**Comment by apoelstra** [comment]
Will hold off on this til #3395 because I really want green CI before merging any CI fixes. (For other stuff I'm willing to trust my own local CI to make sure we're MSRV compatible.)

**Review by apoelstra — APPROVED** [review]
ACK b8dab65c3dbfb347f2dd3504c0bc935cc871729f successfully ran local tests; nice

**Comment by apoelstra** [comment]
I guess I should one-ACK merge this since it's CI only.

**Comment by tcharding** [comment]
Yes please

**Comment by apoelstra** [comment]
In the queue. Two PRs ahead of it, may be a while.

**Comment by tcharding** [comment]
No stress, its about time to get the kids up over here anyways, I'll be busy cooking pancakes for bit :)

**Pull request merged**
